weld/jpa/src/main/java/org/jboss/as/weld/services/bootstrap/WeldJpaInjectionServices.java

```diff
@@ -134,6 +134,9 @@ private String getScopedPUName(final DeploymentUnit deploymentUnit, final String
         PersistenceUnitMetadata scopedPu;
         scopedPu = PersistenceUnitSearch.resolvePersistenceUnitSupplier(deploymentUnit, persistenceUnitName);
         if (null == scopedPu) {
+            if (persistenceUnitName.equals("")) {
+                throw WeldLogger.ROOT_LOGGER.couldNotLoadPersistenceXmlFromInvalidLocation();
```

Comment:
Do we know this means an invalid location? Can we end up here just because there's no persistence.xml at all?

Reply:
It is for both cases. Wrong location and no persistence.xml at all..
I will make it only for wrong location

Reply:
It's better to make two different cases: The first one checks if not exists at all and throw warning, the second one checks if exist in wrong location and also throws warning?
OR 
Make one case which cover both of them and warn the developer that "perstistence.xml not exist or its location is wrong"
?

Reply:
Ok, thank you, I will make it this way